python/dgl/data/flickr.py

```diff
@@ -0,0 +1,163 @@
+"""Flickr Dataset"""
+import os
+import json
+import numpy as np
+import scipy.sparse as sp
+from .. import backend as F
+from ..convert import from_scipy
+from ..transforms import reorder_graph
+from .dgl_dataset import DGLBuiltinDataset
+from .utils import generate_mask_tensor, load_graphs, save_graphs, _get_dgl_url
+
+
+class FlickrDataset(DGLBuiltinDataset):
+    r"""Flickr dataset for node classification from `GraphSAINT: Graph Sampling Based Inductive
+    Learning Method <https://arxiv.org/abs/1907.04931>`_
+
+    The task of this dataset is categorizing types of images based on the descriptions and common
+    properties of online images.
+
+    Flickr dataset statistics:
+
+    - Nodes: 89,250
```

Comment:
This is a minor issue, but it might be better to be consistent for whether to have an empty space after the comma.

Reply:
I see that the other dataset's source code does not have a blank space after the comma when presenting the dataset's statistics.